mlflow/ml-package-versions.yml

```diff
@@ -209,7 +209,7 @@ onnx:
       pip install dist/*manylinux*.whl
 
   models:
-    minimum: "1.14.1"
+    minimum: "1.15.0"
```

Comment:
can we revert version updates?

Reply:
Yes, should we only update when releasing?
Cuz by default they're updated when we run the script

Reply:
You can use the `--skip-yml` flag.